warehouse/archiver.go

```diff
@@ -109,15 +112,22 @@ func usedRudderStorage(uploadMetdata []byte) bool {
 }
 
 func archiveUploads(dbHandle *sql.DB) {
+	pkgLogger.Infof(`Started archiving for warehouse`)
 	sqlStatement := fmt.Sprintf(`SELECT id,source_id, destination_id, start_staging_file_id, end_staging_file_id, start_load_file_id, end_load_file_id, metadata FROM %s WHERE ((metadata->>'archivedStagingAndLoadFiles')::bool IS DISTINCT FROM TRUE) AND created_at < NOW() -INTERVAL '%d DAY' AND status = '%s'`, warehouseutils.WarehouseUploadsTable, uploadsArchivalTimeInDays, ExportedData)
 
 	rows, err := dbHandle.Query(sqlStatement)
+	defer func() {
+		if err != nil {
+			pkgLogger.Errorf(`Error occurred while archiving for warehouse uploads with error: %v`, err)
+			stats.NewTaggedStat("warehouse.archiver.uploadAborted", stats.CountType, stats.Tags{}).Count(1)
```

Comment:
rename stat as something like archiveFailed instead of uploadAborted?
just to distinguish with actual upload aborted stats

Reply:
Done.